pkg/apis/k8s/v1alpha1/zz_generated.openapi.go

```diff
@@ -25,12 +25,12 @@ import (
 
 func GetOpenAPIDefinitions(ref common.ReferenceCallback) map[string]common.OpenAPIDefinition {
 	return map[string]common.OpenAPIDefinition{
-		"./pkg/apis/k8s/v1alpha1.ContainerSpec": schema_pkg_apis_k8s_v1alpha1_ContainerSpec(ref),
-		"./pkg/apis/k8s/v1alpha1.Password":      schema_pkg_apis_k8s_v1alpha1_Password(ref),
-		"./pkg/apis/k8s/v1alpha1.Redis":         schema_pkg_apis_k8s_v1alpha1_Redis(ref),
-		"./pkg/apis/k8s/v1alpha1.RedisList":     schema_pkg_apis_k8s_v1alpha1_RedisList(ref),
-		"./pkg/apis/k8s/v1alpha1.RedisSpec":     schema_pkg_apis_k8s_v1alpha1_RedisSpec(ref),
-		"./pkg/apis/k8s/v1alpha1.RedisStatus":   schema_pkg_apis_k8s_v1alpha1_RedisStatus(ref),
+		"github.com/amaizfinance/redis-operator/pkg/apis/k8s/v1alpha1.ContainerSpec": schema_pkg_apis_k8s_v1alpha1_ContainerSpec(ref),
```

Comment:
If I am not mistaken newer versions of operator-sdk replaced absolute paths with those relative dot-style paths.
Please try regenerating them with operator-sdk version v0.10.0

If you experience issues like this https://github.com/operator-framework/operator-sdk/issues/1854
try running:
```
GOROOT=$(pwd) operator-sdk generate k8s --verbose && GOROOT=$(pwd) operator-sdk generate openapi --verbose
```

Reply:
I've tried to generate using operator-sdk v0.10.0 but with no luck, the paths are still absolute. I don't know how to fix it.